mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -1145,6 +1145,46 @@ def Rock_AccelGemmOp:
   let hasVerifier = 1;
 }
 
+// blockwise_broadcasting_reduction
+def Rock_BlockwiseBroadcastReduceOp:
+    Rock_Op<"blockwise_broadcast_reduce">,
+    Arguments<(ins MemRefOf<GemmOutputTypes>:$input,
+                   MemRefOf<LdsBufferTypes>:$workspace_buffer,
+                   MemRefOf<GemmOutputTypes>:$output,
+                   IndexAttr:$axis,
+                   ReduceMethodAttr:$reduceMethod,
+                   TransformMapArrayAttr:$inputRegView,
+                   I32Attr:$blockSize
+                   )> {
+  let summary = "Blockwise reduce operation with output broadcasting";
+  let description = [{
+    The `rock.blockwise_broadcast_reduce` op does reduce at a blockwise level and
+    to fill all elements of the input tensor with reduced values. Note that both input/output
+    tensors live in registers.
+
+    $input tensor resides in registers in threads.
+
+    $inputRegView should be register {tid, iter}
```

Comment:
"should be a coordinate transform taking `{tid, iter}` (where `iter` is the register number in `$input` and `tid` is the ID of the lane within the workgroup and producing indices into a virtual tensor `{d0, ..., dR, ... dN} which is spread across the lanes of the workgroup."

Reply:
done